firmware/application/apps/ui_recon.cpp

```diff
@@ -354,16 +367,21 @@ void ReconView::focus() {
 }
 
 ReconView::~ReconView() {
-    // Save recon config.
+    recon_stop_recording();
+    delete record_view;
```

Comment:
Prefer a unique_ptr class member instead of raw allocations.

Reply:
Here you won't need to delete anything because the unique_ptr will delete for you as your class is destroyed.